source/documentation/virtual_machine_management_guide/topics/Removing_an_Affinity_Label.adoc

```diff
@@ -2,11 +2,11 @@
 :_content-type: PROCEDURE
 :_content-type: PROCEDURE
 [id="Removing_an_Affinity_Label"]
-= Removing an Affinity Label
+= Deleting an Affinity Label
 
 You can remove affinity labels only from the details view of a cluster.
```

Comment:
You can remove affinity labels only from the details view of a cluster after it was deleted from each entity.